### PR TITLE
Chores: Moved JSON sources to separate files

### PR DESCRIPTION
**Pull request by TorsteinHonsi**
It's better structured and easier to maintain this way.

Also added JSON source for the new forecast demo.

### CODE REVIEW

**Review by bre1470 — COMMENTED** [review]
I get 9 errors in my setup:

``` Shell
Firefox 115.0 (Mac OS 10.15) Highcharts Map drilldown with disabled animation FAILED
        Promise rejected during "Map drilldown with disabled animation": Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.topo.json" missing in JSONSources (trying to fetch)
Firefox 115.0 (Mac OS 10.15) Highcharts Map drilldown with zooming animation FAILED
        Promise rejected during "Map drilldown with zooming animation": Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.topo.json" missing in JSONSources (trying to fetch)
Firefox 115.0 (Mac OS 10.15) Highcharts Map drilldown with disabled zooming animation FAILED
        Promise rejected during "Map drilldown with disabled zooming animation": Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.topo.json" missing in JSONSources (trying to fetch)
Firefox 115.0 (Mac OS 10.15) Highcharts Map drilldown animation for GeoJSON maps FAILED
        Promise rejected during "Map drilldown animation for GeoJSON maps": Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.geo.json" missing in JSONSources (trying to fetch)
Firefox 115.0 (Mac OS 10.15) Highcharts #20886, Map drilldown datalabel mouse tracking FAILED
        Promise rejected during "#20886, Map drilldown datalabel mouse tracking": Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.geo.json" missing in JSONSources (trying to fetch)
Firefox 115.0 (Mac OS 10.15) Highcharts Zoom in/out with padding, panning in both directions. FAILED
        Promise rejected during "Zoom in/out with padding, panning in both directions.": Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.topo.json" missing in JSONSources (trying to fetch)
Firefox 115.0 (Mac OS 10.15) Highcharts Recommend map view for map chart. FAILED
        Promise rejected during "Recommend map view for map chart.": Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.topo.json" missing in JSONSources (trying to fetch)
Firefox 115.0 (Mac OS 10.15): Executed 140 of 285 SUCCESS (0 secs / 14.664 secs)
Firefox 115.0 (Mac OS 10.15) Highcharts #19980/pattern-fill/geojson FAILED
        Error downloading or processing data: Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.geo.json" missing in JSONSources (trying to fetch)
        Debug: http://utils.highcharts.local/samples/#test/unit-tests/color/pattern-fill
        Expected: true
        Actual: false
        @samples/unit-tests/color/pattern-fill/demo.js:801:20

Firefox 115.0 (Mac OS 10.15) Highcharts #19980/pattern-fill/topojson FAILED
        Error downloading or processing data: Sample error, URL "https://code.highcharts.com/mapdata/custom/world-continents.topo.json" missing in JSONSources (trying to fetch)
        Debug: http://utils.highcharts.local/samples/#test/unit-tests/color/pattern-fill
        Expected: true
        Actual: false
        @samples/unit-tests/color/pattern-fill/demo.js:857:20
```

**Comment by TorsteinHonsi** [comment]
@bre1470 What command is that? I'm running `npx gulp test` without errors, my setup being Firefox 115.0 (Mac OS 10.15) like yours. However, I got those exact errors before the last commit (no not recreate window.JSONSources...). Did you remember to pull the latest changes?

File: test/karma-conf.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const yaml = require('js-yaml');
 const path = require('path');
 const os = require('os');
 const { getLatestCommitShaSync } = require('../tools/libs/git');
+const alias = require('../samples/data/json-sources/alias.json');
```
Comment (bre1470) [inline test/karma-conf.js:8]: It might be clearer to have the file called `index.json`.

**Review by bre1470 — APPROVED** [review]
Yes, that was it. Somehow it did not had the latest commit, even though I did fetch/pull on master before. 

Looks good! 👍

**Pull request merged**